lib/hanami/commands/generate/action.rb

```diff
@@ -234,14 +234,16 @@ def spec_root
         end
 
         def relative_action_path
-          path = Pathname.new('.').join('..', '..', '..')
-          path = path.join('..') if environment.container?
+          nestings = ['..'] * @controller_pathname.count('/')
```

Comment:
@berfarah the first part of `relative_action_path` is identical to `relative_view_path`. Shall we consider to extract a method to be used by both of these consumers?